lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -83,12 +104,28 @@ dampening_suppress_time:
 
 default_information:
   kind: boolean
-  config_get_token_append: '/^default-information originate$/'
-  config_set_append: '<state> default-information originate'
-  default_value: false
+  cli_nexus:
+    config_get_token_append: '/^default-information originate$/'
+    config_set_append: ' <state> default-information originate'
+    default_value: false
+  cli_ios_xr:
+    # IOS XR does not have default-information under address-family,
+    #   so we must replace config get/set
+    # Perhaps better to fix in ruby code?
```

Comment:
If this is a 'bgp_af' property that XR BGP doesn't support under any address families, then this property should be excluded on XR. (I guess on XR, from the comment, that this command is configured in some other scope? In that context it would then be appropriate to have a future work item to add a version of this property under the appropriate object for that scope.)